conda_concourse_ci/execute.py

```diff
@@ -690,23 +690,23 @@ def graph_to_plan_with_jobs(
 
 def build_automated_pipeline(resource_types, resources, remapped_jobs, folders, order):
     # resources to add
-    deployment_approval = OrderedDict({
```

Comment:
but what about python 2.7 support!?!?!?!?!

Reply:
I got rid of it, don't worry